src/shared/models/money.model.ts

```diff
@@ -1,13 +1,35 @@
+import { STX_DECIMALS } from '@shared/constants';
+import { isUndefined } from '@shared/utils';
 import BigNumber from 'bignumber.js';
 import { Currencies } from './currencies.model';
 
 export interface Money {
   readonly amount: BigNumber;
   readonly symbol: Currencies;
+  readonly decimals: number;
 }
 
-export function createMoney(amount: BigNumber | number, symbol: Currencies): Money {
-  return { amount: new BigNumber(amount), symbol };
+//
```

Comment:
Intended?

Reply:
Force of habit, I used to do this stylistically 👀